medicines/web/src/components/mip/index.tsx

```diff
@@ -119,17 +121,41 @@ const Mip: React.FC = () => {
     });
   };
 
+  const handleDocTypeCheckbox = async (d: DocType) => {
```

Comment:
Can we use more explicit variable names? `docType` and `allowedDocTypes` for example?

Reply:
Sure